src/components/footers/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import { FooterLinks } from '../molecules/footers/FooterLinks'
+import { navLinks, roles, genres } from '../../common/constants/enums'
+import { BottomIcons } from '../molecules/footers/BottomIcons'
+
+export function Footer() {
+  const generateColumnData = (
+    enumValues: string[],
+    title: string,
+    prefix: string = ''
+  ) => {
+    const items = enumValues.map(value => ({
+      label: value.replace('_', ' '),
+      link: `/${prefix}${value.toLowerCase().replace('_', '-')}`,
+    }))
+    return { title, items }
+  }
+
+  const roleColumnData = generateColumnData(roles, 'ROLE', 'video-list?filter=')
+  const genreColumnData = generateColumnData(
+    genres,
+    'GENRE',
+    'video-list?filter='
+  )
+
+  const transformedNavLinks = navLinks.map(({ route, label }) => ({
+    label,
+    link: route,
+  }))
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        backgroundColor: '#2D2D2D',
```

Comment:
🔧 Instead of harc-coding the color here, I'd use the palette in KurtTheme and then we can access it here. Ensures we're using rh same colors everywhere